src/site/content/en/index.md

```diff
@@ -2,51 +2,275 @@
 layout: 'homepage-next'
 title: 'web.dev'
 intro:
-  eyebrow: 'About web.dev'
-  title: "Let's build the future of the web, together"
+  eyebrow: 'Brought to you by the Chrome DevRel team'
+  title: "Building a better web, together"
   summary: |
-    Take advantage of the latest modern technologies to build amazing web experiences for everyone.
+    Guidance to build modern web experiences that work on any browser
   buttonText: 'Learn more'
   buttonUrl: '/about/'
   image: 'https://web-dev.imgix.net/image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/zrBPJq27O4Hs8haszVnK.svg'
   imageWidth: '333'
   imageHeight: '240'
-featureCard:
-  theme: 'light'
-  title: 'Watch the progress we made together on the web.'
-  eyebrow: ''
+
+featuredCard:
+  eyebrow: 'Featured'
+  title: 'Celebrate Chrometober with web tricks and treats'
+  background: 'image/SZHNhsfjU9RbCestTGZU6N7JEWs1/8JNKYwBSBG5CtMo2KxkB.png'
   url: 'https://www.youtube.com/watch?v=W9O4CuSUKb8'
-  background: ''
-  video: 'https://web-dev.imgix.net/video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/lIta9ZJEALbfAzJzmiNl.mp4'
-  alt: 'Chrome icon rolls past a title: 2022 Year in Review'
-promoPanel:
-  title: 'Chrome Dev Summit 2021'
-  summary: 'Join the Chrome team for workshops, office hours, and learning lounges taking place throughout November.'
-  primaryButtonUrl: 'https://events.withgoogle.com/chrome-dev-summit-2021/registrations/new/details/'
-  primaryButtonText: 'Request an invite'
-  secondaryButtonUrl: 'https://developer.chrome.com/devsummit/schedule/'
-  secondaryButtonText: 'View schedule'
-  image: 'https://web-dev.imgix.net/image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/KwS4L954TzN3KK3pWmtA.svg'
-  imageWidth: '530'
-  imageHeight: '483'
+  alt: 'Celebrate Chrometober with web tricks and treats'
+  theme: 'light'
+
 pickedCaseStudyUrl: '/mishipay/'
-pickedCaseStudyTheme: 'light'
-courses:
-  title: 'Featured courses'
-  summary: 'Explore our structured learning paths to discover everything you need to know about building for the modern web.'
-caseStudies:
-  title: 'Case studies'
-  summary: 'Learn why and how other developers have used the web to create amazing web experiences for their users.'
-  buttonText: 'See all'
-  buttonUrl: '/tags/case-study/'
-podcasts:
-  title: 'Podcasts and Shows'
-  summary: 'Level up your web development skills by listening to podcasts from Google Developers'
-  buttonText: 'See all'
-  buttonUrl: '/podcasts/'
-patterns:
-  title: 'Patterns'
-  summary: 'A collection of code snippets to help you optimize your web projects.'
-  buttonText: 'See all'
-  buttonUrl: '/patterns/'
+
+news:
+  title: Web platform news
+  subTitle: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+learns:
```

Comment:
should this be named "themes" instead?

Reply:
I have changed from "learns" to "themes"